packages/react-flex/src/Flex.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
```

Comment:
If we're following the same layout as the other packages (which I'd recommend), it should be like this:
```
src/
  Flex.ts           // exports * from ./components/flex/index
  index.ts          // exports * from ./Flex
  components/
    Flex/
      Flex.tsx
      Flex.types.ts
      index.ts      // exports * from Flex and Flex.types
```